tasking/apps.py

```diff
@@ -20,8 +21,6 @@ def ready(self):
         Do stuff when the app is ready
         """
         # set up app settings
-        from django.conf import settings
```

Comment:
I think that it is not safe to move these to the top, as you want the code to run when this app has been registered by Django and not before.  This gives a hint about this idea https://docs.djangoproject.com/en/3.0/ref/applications/#django.apps.AppConfig.ready - but feel free to investigate furtherh.

Reply:
Read more on this. Totally agree with moving the functions into the AppConfig. I'll revert this change.

Reply:
Reverted the change.